TreeDB/cmd/treedb_rag_benchmark/minima_bounded_test.go

```
@@ -0,0 +1,364 @@
+			r.ResourceMeasurement.PeakRSSAvailability = "unavailable"
+			r.ResourceMeasurement.PeakRSSBytes = nil
+			r.ResourceMeasurement.Segments[0].Baseline.PeakRSS.ProcessIdentity = "1:999"
+		}, false},
+		{"missing_baseline_measured_aggregate", func(r *minimaRawBackendEvidence) { r.ResourceMeasurement.Segments[0].Baseline.PeakRSS = nil }, false},
+		{"missing_baseline_unavailable_aggregate", func(r *minimaRawBackendEvidence) {
+			r.ResourceMeasurement.Segments[0].Baseline.PeakRSS = nil
+			r.ResourceMeasurement.PeakRSSAvailability = "unavailable"
+			r.ResourceMeasurement.PeakRSSBytes = nil
+		}, true},
+		{"doctored_envelope", func(r *minimaRawBackendEvidence) {
+			copy := r.ResourceMeasurement.Segments[0].Baseline
```

golangci-lint (2.13.2)

[error] variable copy has same name as predeclared identifier

(predeclared)